cinema/serializers.py

```diff
@@ -59,6 +68,7 @@ class MovieSessionListSerializer(MovieSessionSerializer):
     cinema_hall_capacity = serializers.IntegerField(
         source="cinema_hall.capacity", read_only=True
     )
+    tickets_available = serializers.IntegerField(read_only=True)
```

Comment:
The `tickets_available` field is defined but lacks implementation. You need to calculate the actual number of available tickets for a movie session. Consider implementing a method or property to compute this value based on the total capacity and the number of tickets already sold.